lib/MiniDroneBtAdapter.js

```diff
@@ -403,7 +403,10 @@ class MiniDroneBtAdapter extends EventEmitter {
         const manufacturer = peripheral.advertisement.manufacturerData;
         const matchesFilter = localName === this.options.droneFilter;
 
-        const localNameMatch = matchesFilter || DRONE_PREFIXES.some((prefix) => localName.indexOf(prefix) >= 0);
```

Comment:
@fetherston It turns out what's happening is there are other (non-matching) devices in my house that have `localName` undefined. This causes an NPE on the `.indexOf` call. When the proper peripheral is actually found, `localName` is indeed defined and matches `Mambo_`. So, this is a general problem for any situation in which one tries to connect to the drone in the presence of other devices not broadcasting a `localName`. Thoughts?